config/application.yml.template

```diff
@@ -1,51 +0,0 @@
-# Add configuration values here, as shown below.
```

Comment:
So technically everyone should be adding to the template as well dummy values when a new ENV key is introduced.